justfile

```diff
@@ -0,0 +1,23 @@
+init: purge
+    mkdir build
+
+build:
+    cd build; cmake -DCMAKE_BUILD_TYPE=Release ..; make
+
+purge:
+    rm -rf build
+
+test: build
```

Comment:
Is this file intended to be checked in? The test commands seem specific to this change. If not, maybe gitignore it?

Reply:
Thanks. For the `justfile`, I'll probably just stick to my plan of keeping it in the git history while developing, removing it before merging, and then squash merge at the end to keep the history clean.